### PR TITLE
Updates for 2.5

### PR DESCRIPTION
**Pull request by mantognini**
## Checklist
- [x] Generate new documentations
  - [x] Create new Doxygen file
  - [x] Update version number
  - [x] Build documentation with the new Doxygen file
  - [x] Move the generated documentation into a new sub directory
  - [x] Update the link to the new documentation on the learn page
  - [x] Add the link to the old documentation to the bottom of the learn page
  - [x] Update `$latest` in `documentation/header(-fr)?.php`
- [ ] Update tutorials
  - [x] Add a sub directory for new version
  - [x] Update the link to the new tutorial on the learn page
  - [x] Add the link to the old tutorial to the bottom of the learn page
  - [x] Update `$latest` in `tutorials/header(-fr)?.php` as well as `$version` and `$full_version` in `tutorials/<LATEST>/header(-fr)?.php`.
  - [ ] Check that the tutorials no longer refer primarily to deprecated features.
  - [x] Updates for SFML/SFML#1335 and SFML/SFML#1361.
  - [x] Updates for #105 and #102.
- [ ] Check if the FAQ needs updating
- [x] Update the Download page
  - [x] Add a sub directory for the new version
  - [x] Update the link and text to new version on the download page
  - [x] Add link to the old version on the old download page
  - [x] Update download package sizes
- [x] Update the Changelog
  - [x] Add a new section for the latest changes
  - [x] Add a link to the new section at the top
- [ ] Update nginx redirections for:
  - [ ] http://www.sfml-dev.org/documentation/ -> http://www.sfml-dev.org/documentation/X.Y.Z
  - [ ] http://www.sfml-dev.org/documentation/latest -> http://www.sfml-dev.org/documentation/X.Y.Z
  - [ ] http://www.sfml-dev.org/tutorials/X.Y.Z -> http://www.sfml-dev.org/tutorials/X.Y
  - [ ] http://www.sfml-dev.org/tutorials/latest -> http://www.sfml-dev.org/tutorials/X.Y

### CODE REVIEW

**Comment by eXpl0it3r** [comment]
Cool, thanks for starting this! 😊
I made an attempt at starting changelog yesterday but haven't gotten far. Unless someone wants to pick it up, I'm gonna continue.

**Comment by Foaly** [comment]
I could help writing the changlog if you want.

**Comment by eXpl0it3r** [comment]
If you want to do it, no problem! 🙂 
More time for me to spend on other preparation and testing.

Guess you can simply open a PR with your changes based of the 2.5 branch.

**Comment by Foaly** [comment]
I've sent a PR with the changelog updates: https://github.com/SFML/SFML/pull/1389
Once this is reviewed and accepted I will try to convert it into HTML for the website (if wanted).

File: tutorials/2.5/window-opengl.php [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+{
+    // create the window (remember: it's safer to create it in the main thread due to OS limitations)
+    sf::Window window(sf::VideoMode(800, 600), "OpenGL");
+
+    // deactivate its OpenGL context
+    window.setActive(false);
+
+    // launch the rendering thread
+    sf::Thread thread(&amp;renderingThread, &amp;window);
+    thread.Launch();
```
[the remark this answers is not in the document]
Comment (mantognini) [reply tutorials/2.5/window-opengl.php]: Thanks! It's now amended.

**Comment by eXpl0it3r** [comment]
Added the download pages. Also did this to the download page, hopefully that reduced the amount of certain questions:



This paragraph needs translation:

> It's recommended to use the SFML version from your package manager (if recent enough) or build from source to prevent incompatibilities.

And I dropped TDM 64bits support. The only reason this compiler is still supported is for all the Code::Blocks users and Code::Blocks doesn't ship a 64-bits version, so it doesn't make sense to maintain that on our side.

**Comment by mantognini** [comment]
Il est recommandé d'utiliser la version de SFML fournie par votre gestionnaire de paquets (si elle est suffisamment récente), ou de compiler SFML depuis ses sources pour éviter des problèmes d'incompatibilité.

**Pull request merged**